cips/cip-start-times.md

```diff
@@ -0,0 +1,63 @@
+- `TargetInterval` is part of `BlockParams` within `ConsensusParams`. A `TargetInterval` of zero signifies that this mechanism is disabled. The `TargetInterval` CAN be set in a hard fork upgrade when all nodes are running the latest version, activating the new timing mechnanism.
+- The `TargetInterval` once activated will be set to 12 seconds. `TargetInterval` can not be modified via governance and SHOULD only change in major version upgrades.
+- `TimeoutCommit` will be ignored when this mechanism is active.
+
+To help develop an intuition on what this proposal changes, if we were to plot all the start times of each validator in the network in order, assuming equal weighting, the start times of all nodes for the **next height** will be within the "middle third of start times" of each other. This has a converging effect even in cases where consensus takes longer than the `TargetInterval` itself.
+
+## Parameters
+
+| Parameter     | Proposed value | Description                                                                                                                | Changeable via Governance |
+|---------------|---------|------------------------------------------------------------------------------------------------------------------------|---------------------------|
+| BlockParams.TargetIntervalMS | 12000 milliseconds (12 seconds)  | The target interval between start times in milliseconds                                                            | false                     |
+
+## Rationale
+
+Given the asynchronous nature of networks, it's difficult to coordinate around the commit times as it's difficult to predict how long it will take to commit a block. What is more predictable is to coordinate around the start times i.e. the time that the proposer begins to propose a block. By including every validators start time in their votes, we have a relatively robust reference point for all nodes. The start time of the next height.
+
+## Backwards Compatibility
+
+Enabling of this mechanism must be coordinated amongst all consensus nodes in the network. It should tie in with a major upgrade. The changes themselves can be made in a backwards compatible manner to `celestia-core` by having them be disabled by default.
+
+## Reference Implementation
+
+To be filled out.
+
+## Security Considerations
+
+This modification changes the time located in the block to be the stake-weighted median of the start times of 2f + 1 validators from the previous block. `TargetInterval` is added to this as an approximation of when the current block was actually started. This is different from the how the current block time is calculated, however, this shift in the times is not expected to have any downstream impact.
+
+Similar with traditional BFT time, a malicious cabal greater than 1/3 can manipulate the block time to be any time in the future.
```

Comment:
[Question to understand more]

how far can 1/3rd of the network push time in the future? Indefinitely or there is an upper bound related to the target interval mechanism